pandas/core/window/rolling.py

```diff
@@ -1706,10 +1707,15 @@ def validate(self):
 
     def _validate_monotonic(self):
         """
-        Validate on is_monotonic.
+        Validate monotonic (increasing or decreasing).
         """
-        if not self._on.is_monotonic:
-            formatted = self.on or "index"
+        if (
+            not self._on.is_monotonic_increasing
```

Comment:
can you write this as not(self._on.is_monotonic_increasing or self._on.is_monoto....)

as more clear